src/data/faq.json

```diff
@@ -573,6 +573,15 @@
                             "For more information on the STIKO vaccination recommendations and all about COVID-19 and vaccination, see <a href='https://www.rki.de/DE/Content/Infekt/Impfen/ImpfungenAZ/COVID-19/COVID-19.html' target='_blank' rel='noopener noreferrer'>RKI - Impfungen A - Z - COVID-19 und Impfen (German)</a>."
                         ]
                     },
+                    {
+                        "title": "2/1 vaccination certificates",

```

Comment:
Suggest to capitalize "Vaccination".

It's the only article which starts with a digit, so there is no comparative article to know what the convention is.
Articles which start with letters all have their first word capitalized in English though.